src/components/Comments/commentsSlicer.tsx

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+// eslint-disable-next-line import/no-cycle
```

Comment:
Don't disable eslint, try to solve the error instead. Fix it everywhere

Reply:
eslint-disable was already there when I've forked the repo

Reply:
But you are currently working on a project and taking responsibility. You didn't even read the essence of those comments, maybe after the development, some comments are not relevant at all